deploy/api.yaml

```diff
@@ -0,0 +1,157 @@
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVQuestionFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+  /answer:
+    post:
+      parameters:
+        - $ref: "#/components/parameters/SessionHeader"
+      responses:
+        "201":
+          description: "201 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Answer"
+        "400":
+          description: "400 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+        "500":
+          description: "500 response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+      x-amazon-apigateway-request-validators: "Validate body"
+      x-amazon-apigateway-integration:
+        type: "aws_proxy"
+        httpMethod: "POST"
+        uri:
+          Fn::Sub: "arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${KBVAnswerFunction.Arn}/invocations"
+        passthroughBehavior: "when_no_match"
+components:
+  parameters:
+    SessionHeader:
+      name: session-id
+      in: header
+      description: A UUID generated by the KBV API to act as a primary key for the KBVSessionTable in DynamoDB
+      required: true
+      schema:
+        type: string
+  schemas:
+    Session:
+      type: "object"
+      properties:
+        session-id:
+          type: "string"
+    Error:
+      title: "Error Schema"
+      type: "object"
+      properties:
+        message:
+          type: "string"
+    Question:
+      type: "object"
+      properties:
+        questionID:
+          type: "string"
+        text:
+          type: "string"
+        tooltip:
+          type: "string"
+        answerHeldFlag:
+          type: "string"
+        answerFormat:
+          type: "object"
+          $ref: "#/components/schemas/AnswerFormat"
+    AnswerFormat:
+      type: "object"
+      properties:
+        identifier:
+          type: "string"
+        fieldType:
+          type: "string"
+        answerList:
+          type: array
+          items:
+            type: "string"
+    Answer:
+      type: "object"
+      properties:
+        questionId:
+          description: The unique identifier for the question
+          maxLength: 6
+          type: "string"
+        answer:
```

Comment:
For the benefit of people not familiar with the Experian API, answers are returned as text, without an ID, which is why the `Answer` has a `questionId` but a `answer` string.

Reply:
I can confirm the above statement is correct.